plugins/statistics.koplugin/main.lua

```diff
@@ -284,7 +296,7 @@ function ReaderStatistics:getCurrentStat()
     }
 end
 
-local function generateReadBooksTable(title, dates)
+function ReaderStatistics:generateReadBooksTable(title, dates)
```

Comment:
When I use:
`local function generateReadBooksTable(title, dates)`
I get error
`./luajit: plugins/statistics.koplugin/main.lua:199: attempt to call method 'generateReadBooksTable' (a nil value)
stack traceback:
	plugins/statistics.koplugin/main.lua:199: in function 'callback'
	frontend/ui/widget/touchmenu.lua:567: in function 'action'
	frontend/ui/uimanager.lua:466: in function '_checkTasks'
	frontend/ui/uimanager.lua:612: in function 'handleInput'
	frontend/ui/uimanager.lua:706: in function 'run'
	./reader.lua:188: in main chunk
	[C]: at 0x00404750
`
while I go to Statistics -> Time range -> Last Week.
The same error in other time range statistics.
How to change this line?
`kv_pairs = self:generateReadBooksTable("", self:getDatesFromAll(7, "daily_weekday")),`